assets/src/story-embed-block/block/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+// TODO: Replace with web stories icon.
```

Comment:
Can we fix this now?

Reply:
We don't have such an icon / logo yet. Needs UX to provide one for us.

Mentioned as a todo in the PR description:

> UX: Provide custom block icon

Also mentioned on the issue itself:

https://github.com/google/web-stories-wp/issues/1118#issuecomment-624894364

Reply:
Can we use the story icon we are using for the CPT for now?